README.md

```diff
@@ -1,95 +1,190 @@
+
+* **No Lock-In:** Specifically made for beginners and to provide fast bootstrap for new projects, create-cycle-app doesn't have the ambition to be __the__ tool for working with Cycle.js projects. With that in mind, it's easy to leave `create-cycle-app` defaults and follow your own steps, by running `npm run take-off-training-wheels`.
+
+## Why Use This?
+
+**If you’re getting started** with Cycle.js, use `create-cycle-app` to automate the build of your app. There is no configuration file, and `cycle-scripts-<flavorName>` is the only extra build dependency in your `package.json`. Your environment will have everything you need to build a Cycle.js app.
+
+**If you’re a power user** simply use it as a boilerplate generator, by passing in your own flavor.
+
+
+## Getting Started
+
+### Installation
+
+Installing globally provides a create-cycle-app command for creating new projects.
+
+```sh
+$ npm install --g create-cycle-app
+```
+
+**We recommend** to use Node >= 6 and npm >= 3 for faster installation speed and better disk usage. You can use a node version manager(i.e [nodenv](https://github.com/nodenv/nodenv), [nvm](https://github.com/creationix/nvm), [n](https://github.com/tj/n)) to easily switch Node versions among different projects.
+
+### Creating an App
+
+To create a new cycle.js app, run:
+
+```
+$ create-cycke-app my-awesome-cycle-app
+$ cd my-awesome-cycle-app
+```
+
+It will create a directory called `my-awesome-cycle-app` inside the current folder.
+Inside that directory, it will generate the following initial project structure* and install the required dependencies.
 
-- [cycle-scripts-es-browserify](./cycle-scripts-es-browserify/): ES6 (babel) + Browserify
-- [cycle-scripts-ts-browserify](./cycle-scripts-ts-browserify/): TypeScript + Browserify
-- [cycle-scripts-es-webpack](./cycle-scripts-es-webpack/): ES6 (babel) + Webpack
-- [cycle-scripts-ts-webpack](./cycle-scripts-ts-webpack/): TypeScript + Webpack
+```
+my-awesome-cycle-app/
+├── node_modules/
+├── puplic/
+│   ├── favicon.ico
+│   └── index.html
+├── src/
+│   ├── app.js
+│   ├── app.test.js
+│   └── index.js
+└── package.json
+```
+
+_* Generated structure and files may change depending on the flavor being used, The above structure hold true for any of the 4 core flavors._
+
+No configuration or complicated folder structures, just the files you need to build your cycle app.
+Once the installation is done, you can run the following commands from within the project folder:
+
+### `npm start`
+
+Runs the app in development mode by starting the development server.
+
+Core flavors come with [ live | hot module ] reload so that the page will reload if you make edits.
```

Comment:
Maybe we should omit this part about "hot module reload/replacement", as we didn't ship this, yet.